charts/umbrella/Chart.yaml

```diff
@@ -28,7 +28,7 @@ sources:
   - https://github.com/eclipse-tractusx/tractus-x-umbrella
 
 type: application
-version: 2.14.1
+version: 2.14.2
```

Comment:
please merge main and remove the change of the version, updates to the docs directory don't require to increment the chart version